R/modify_side.R

```diff
@@ -0,0 +1,26 @@
+#' modify a side to update it with new arguments
+#' 
+#' @param object gsplot object
+#' @param side sides to modify with extra parameters
+#' @param \dots user arguments to passed through
+#' 
+#' @keywords internal
+#' @examples
+#' gs <- gsplot() %>%
+#'    lines(c(0,10), c(0,10))
+#' gs <- gsplot:::modify_side(gs, c(1,2), xlim=c(0,12))
+#' 
+modify_side <- function(object, side, ...) {
```

Comment:
preference on having side before arguments in these signatures? Most of the functions have side last